src/main/java/io/vertx/core/http/WebSocketBase.java

```diff
@@ -90,6 +91,15 @@
    */
   String subProtocol();
 
+  /**
+   *  Returns the HTTP response headers during the websocket connection handler.
+   *  <p/>
+   *  After the connection handler has completed the response headers will be null.
```

Comment:
_After the completion handler callback has completed the response headers will be {@code null}_